src/snyk/common/languageServer/lsExecutable.ts

```diff
@@ -0,0 +1,31 @@
+import path from 'path';
+import { Checksum } from '../../cli/checksum';
+import { SupportedPlatform } from '../../cli/supportedPlatforms';
+import { Platform } from '../../common/platform';
+
+export class LsExecutable {
+  // If values updated, `.vscodeignore` to be changed.
+  public static filenameSuffixes: Record<SupportedPlatform, string> = {
+    linux: 'snyk-ls-linux',
```

Comment:
It's a bit more complicated... please have a look here: https://github.com/snyk/vscode-extension/compare/main...feat/ROAD-1054_manage_ls_binary_automatically#diff-3780f20a3e492c878059b07f3512993edde0e01b86fc3a2f9f4b5fba8bf821f1R1